templates/patterns/file_translation/file_translation.ui_patterns.yml

```diff
@@ -48,3 +55,9 @@ file_translation:
       label: "More information"
       description: "Informative text about the available translations."
       preview: "Looking for another language which is not on the list? Find out why."
+    highlighted:
```

Comment:
We agreed with @ademarco we won't add this style of property and instead we will use the same "Badges" approach as we use in other patterns